frontend/cypress/.env.e2e.sample

```diff
@@ -5,4 +5,4 @@ CYPRESS_OKTA_SECRET=
 
 # Smarty streets secrets
 SMARTY_AUTH_ID=
-SMARTY_AUTH_TOKEN=
+SMARTY_AUTH_TOKEN=
```

Comment:
what changed here?

Reply:
It was only a whitespace change, but for some reason Git won't drop the file from this PR 🤷

Reply:
There may be a newline character, a null character, or something else that isn't human-detectable. Git is weird.